README.md

```diff
@@ -156,6 +156,10 @@ See the provided ready-to-go scripts under the [examples/](./examples) directory
 
 Please read [Contributing](CONTRIBUTING.md)
 
+## Bug reports
+
+Please read [how to create a bug report](https://github.com/huawei-noah/SMARTS/wiki/Bug-Report-Format) and then open an issue [here](https://github.com/huawei-noah/SMARTS/issues).
```

Comment:
I think the link is `https://github.com/huawei-noah/SMARTS/wiki/How-To-Make-a-Bug-Report`?

Reply:
Right, good catch I had changed the name of the page.